tstdc/ffi/build.rs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2022 The MobileCoin Foundation
+//! Builds the FFI function bindings for tstdc (trusted standard C) of the
+//! Intel SGX SDK
+
+use bindgen::Builder;
+use cargo_emit::{rustc_link_lib, rustc_link_search};
+
+fn main() {
+    let sgx_library_path = mc_sgx_core_build::sgx_library_path();
+    rustc_link_lib!("sgx_tstdc");
```

Comment:
This needs to be `static=sgx_tstdc`

Reply:
Looking at https://doc.rust-lang.org/rustc/command-line-arguments.html#option-l-link-lib 
> If the kind is not specified in the link attribute or on the command-line, it will link a dynamic library if available, otherwise it will use a static library

I have no problem being explicit about the static though.

Reply:
added `static` qualifier in bb4305b